web/src/routes/RepositoryDetails/Settings/RepoEventExpiry.tsx

```diff
@@ -28,6 +44,13 @@ export default function RepoEventExpiry(props: RepoEventExpiryProps) {
         required
         validated={valid}
       />
+      {valid == ValidatedOptions.error ? (
```

Comment:
nit: You can use <[Conditional](https://github.com/quay/quay/blob/master/web/src/components/empty/Conditional.tsx#L2)> component instead of ternary operator